### DISCUSSION TITLE
v1.33.x 版本 无法使用telnet连接终端主机

### DISCUSSION THREAD

**Discussion by spiketang** [electerm/electerm#3010] [Q&A]
试了windows11 和 macos ventura 13.4.1版本 装了1.33.6后,无法连接主机；
1、macOS之前是装的老版，连接没问题，升级1.33.6后无法连接，报错说，因为不记得之前的版本号是多少，又退回装了1.26.0，能够连接终端了；
2、windows11上同样报错，

![image](https://github.com/electerm/electerm/assets/3364115/3ccb81ad-a99b-4c01-abfd-4236e1d9956d)

3、经过测试，发现1.33.0和1.33.6版本无论 是win11还是macos arm版本的都无法连接终端；只有1.32及以下的版OK

**Comment by wpf973** [comment]
electerm-1.37.20-win-x64也不行的。求解决

**Comment by zxdong262** [comment]
感谢反馈，将在下个版本修复

**Comment by wpf973** [comment]
更新到1.37.46版本了 可以显示登录界面了 无法输入任何字符。锐捷交换机
